src/shippingservice/go.mod

```diff
@@ -1,43 +1,40 @@
 module github.com/GoogleCloudPlatform/microservices-demo/src/shippingservice
 
-go 1.21
-
-toolchain go1.23.0
```

Comment:
Is the issue that you removed the toolchain line here, so it defaults to trying to find a toolchain matching the Go version on line 3 (`1.23`) which is invalid, maybe?: https://github.com/golang/go/issues/62278#issuecomment-1698829945

Reply:
No 😉 

This is caused by https://github.com/golang/go/issues/68971#issuecomment-2299062164

The GitHub workflows are using Go 1.22.2.

I will bump Go version in GitHub Actions.

Reply:
https://github.com/GoogleCloudPlatform/microservices-demo/pull/2716/commits/660990d0ac7d68680de4901074e46c5690a799b9